vendor/doctrine/cache/.travis.yml

```diff
@@ -26,6 +26,7 @@ before_install:
     - if [[ $TRAVIS_PHP_VERSION = 7.* ]] ; then pecl config-set preferred_state beta; echo yes | pecl install apcu; fi;
     - if [[ $TRAVIS_PHP_VERSION != 'hhvm' ]]; then phpenv config-add ./tests/travis/php.ini; fi;
 
+
```

Comment:
will be good to revert this. here's a quick way to revert a file like this which "sidesteps" git:
git checkout master
cp vendor/doctrine/cache/.travis.yml ../
git checkout new_erx
cp ../.travis.yml  vendor/doctrine/cache/.travis.yml
git commit -a -m "reverted the vendor/doctrine/cache/.travis.yml file to original"